FreeRTOS_TCP_IP.c

```diff
@@ -1153,9 +1150,12 @@
                 #if ( ipconfigUSE_IPv6 != 0 )
                     if( ( pxSocket != NULL ) && ( pxSocket->bits.bIsIPv6 != pdFALSE_UNSIGNED ) )
```

Comment:
The v4 logic on line 1158 verifies that `pxIPHeader != NULL` prior to dereferencing it at line 1160.  Should we do the same at line 1151, e.g.:

```c
if( ( pxSocket != NULL ) && 
    ( pxSocket->bits.bIsIPv6 != pdFALSE_UNSIGNED ) && 
    ( pxIPHeader_IPv6 != NULL ) )
```

Reply:
Thanks Gary, I added the extra test for `pxIPHeader_IPv6` not being NULL.